engineering principles.md

```diff
@@ -1 +1,27 @@
 ## Engineering Principles at TotallyMoney
+
+### Move Fast and Deliver Value
+
+Set yourself up to be able to deliver the most value for the customer with the least amount of friction. It’s about releasing often and without fear, knowing you can always fix any issue or roll back.
+
+#### Practices
+
+- Release often
+- Enable speed through process
+- Quality is a long term enabler
+- Don’t chase perfection
+- There’s nothing simple about simplicity
+- Prioritise the problems that nobody has solved
+- Be aware of services and libraries that can do the heavy lifting
+- Focus on code that enhances the customer experience
+
+#### Benefits
+
+- Knowing it’s ok to release on Fridays
+- Focussing on interesting problems not boilerplate
+- Providing value to the customer
+
+#### Anti-patterns
+
+- Cutting corners to get stuff done
+- Imagining what the customer wants without testing your hypothesis
```

Comment:
"Quality is a long term enabler" negates "Cutting corners to get stuff done"
.. should there be a similar counter-part to "Imagining what the customer wants .."?
A bullet on split-testing to validate hypothesis?